src/test/java/PackageTest.java

```diff
@@ -1,3 +1,63 @@
+import de.cyklon.reflection.entities.OfflinePackage;
+import de.cyklon.reflection.entities.ReflectClass;
+import de.cyklon.reflection.entities.ReflectPackage;
+import de.cyklon.reflection.function.Filter;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import test.TestPackage;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
 public class PackageTest {
-	//TODO cyklon
+
+	private static ReflectPackage pkg = ReflectClass.wrap(PackageTest.class).getPackage();
+	private static OfflinePackage testPkg = pkg.getPackages(Filter.hasSimpleName("test")).stream().findFirst().orElse(null);
```

Comment:
Use the newly created getPackage method here

Reply:
No, because the getPackage method uses Filter.hasName(). However, I use Filter.hasSimpleName()